splunk-otel-android/src/main/java/com/splunk/rum/RumInitializer.java

```diff
@@ -174,7 +176,7 @@ SplunkRum initialize(ConnectionUtil.Factory connectionUtilFactory, Looper mainLo
     private SlowRenderingDetector buildSlowRenderingDetector(Tracer tracer) {
         if (!builder.slowRenderingDetectionEnabled) {
             Log.w(LOG_TAG, "Slow/frozen rendering detection has been disabled by user.");
-            return SlowRenderingDetector.NO_OP;
+            return NoOpSlowRenderingDetector.INSTANCE;
```

Comment:
Was there a problem detected with this or just aesthetic preference?

Reply:
This one's pretty interesting, there wasn't any problem that errorprone detected, but something I noticed when I was doing `instrumentation-api` cleanup: classes defined inside interfaces (like `SlowRenderingDetector`) are always public, even if the interface itself is package-private. We were accidentally making the noop implementation public 😅

Reply:
1000 babies saved!